aten/src/ATen/cuda/CUDAContext.cpp

```diff
@@ -32,14 +32,14 @@ cudaDeviceProp* getDeviceProperties(int64_t device) {
 CUDAStream getStreamFromPool(
   const bool isHighPriority
 , int64_t device) {
-  return detail::CUDAStream_getStreamFromPool(isHighPriority, device);
+  return CUDAStream(detail::CUDAStream_getStreamFromPool(isHighPriority, device));
```

Comment:
wait -- why doesn't this function return a CUDAStream already?

Reply:
The pointer-based API is also needed for the way the translation to Python occurs with streams.